src/sql/workbench/services/jobManagement/browser/jobManagementUtilities.ts

```diff
@@ -40,16 +40,28 @@ export class JobManagementUtilities {
 		return bool ? nls.localize('agentUtilities.yes', "Yes") : nls.localize('agentUtilities.no', "No");
 	}
 
-	public static convertToNextRun(date: string) {
-		return date.includes('1/1/0001') ? nls.localize('agentUtilities.notScheduled', "Not Scheduled") : date;
+	public static convertToNextRun(date: string): string {
+		if (date.includes('1/1/0001')) {
+			return nls.localize('agentUtilities.notScheduled', "Not Scheduled");
+		} else {
+			return date;
+		}
 	}
 
-	public static convertToLastRun(date: string) {
-		return date.includes('1/1/0001') ? nls.localize('agentUtilities.neverRun', "Never Run") : date;
+	public static convertToLastRun(date: string): string {
+		if (date.includes('1/1/0001')) {
+			return nls.localize('agentUtilities.neverRun', "Never Run");
+		} else {
+			return date;
+		}
 	}
 
 	public static setRunnable(icon: HTMLElement, index: number) {
-		icon.classList.remove('non-runnable');
+		let temp = icon.className as unknown;
```

Comment:
This is broken with this logic - just use what I had since that should be the actual fix anyways.